### ISSUE TITLE
Segmantation fault on every command

### ISSUE THREAD

**Issue by MarcelPa** [GothenburgBitFactory/taskwarrior#2054]
I got a problem with taskwarrior recently, it will segfault on every command I try. I had a fairly similar problem some days ago with Electron apps (namely camunda-modeler), which has been resolved with the most recent glibc update. I am on Arch Linux: `Linux mi0061n 4.14.62-1-lts #1 SMP Thu Aug 9 20:50:08 CEST 2018 x86_64 GNU/Linux`

* What command(s) did you run?
task / task sync
* What did you expect to happen?
A list of tasks / syncing of tasks with my taskserver
* What actually happened?
`Segmentation fault (core dumped)`
Using gdb I got the following backtrace, if that helps in any case. I called `gdb task`, ran it and got a backtace on segfault:
```gdb
Program received signal SIGSEGV, Segmentation fault.
0x0000000000000000 in ?? ()
(gdb) bt
#0  0x0000000000000000 in ?? ()
#1  0x00007ffff7940d36 in __gthread_once (__func=0x7ffff7940cd0 <std::locale::_S_initialize_once()>, __once=<optimized out>) at /build/gcc/src/gcc-build/x86_64-pc-linux-gnu/libstdc++-v3/include/x86_64-pc-linux-gnu/bits/gthr-default.h:699
#2  std::locale::_S_initialize () at /build/gcc/src/gcc/libstdc++-v3/src/c++98/locale_init.cc:316
#3  0x00007ffff7940d9e in std::locale::locale() () at /build/gcc/src/gcc/libstdc++-v3/src/c++98/locale_init.cc:250
#4  0x00007ffff793d9bc in std::basic_streambuf<char, std::char_traits<char> >::basic_streambuf (this=<optimized out>) at /build/gcc/src/gcc-build/x86_64-pc-linux-gnu/libstdc++-v3/include/streambuf:470
#5  __gnu_cxx::stdio_sync_filebuf<char, std::char_traits<char> >::stdio_sync_filebuf (__f=0x7ffff76f95c0 <_IO_2_1_stdout_>, this=<optimized out>) at /build/gcc/src/gcc-build/x86_64-pc-linux-gnu/libstdc++-v3/include/ext/stdio_sync_filebuf.h:80
#6  std::ios_base::Init::Init() () at /build/gcc/src/gcc/libstdc++-v3/src/c++98/ios_init.cc:85
#7  0x0000000000433aae in ?? ()
#8  0x000000000057a37d in __libc_csu_init ()
#9  0x00007ffff755e1ae in __libc_start_main () from /usr/lib/libc.so.6
#10 0x0000000000435789 in _start () 
```
* Paste the output of the ```task diag``` command.
diag also segfaults, no information to gain here unfortunaly

I do not fully understand the backtrace, hence I have no idea whether this issue is related to taskwarrior directly. I tried to build the development version (2.6.0 branch), which worked for showing / adding / deleting tasks, but not for syncing. Syncing would also segfault, referring to an error with gnutls libraries (as far as I understood the backtrace).

**Comment by MarcelPa** [comment]
The issue I described occur on Taskwarrior 2.5.1, the version that I get when I install it using pacman on Archlinux from the (official) community repositories.

After digging around for some time now, I figured out to check the bugtracker from the package in Archlinux, I found this bug: https://bugs.archlinux.org/task/59649?project=5&string=task

Apperently, the p11-kit version that is within the testing repositories right now (which I were using) has something to do with it.
Long story short, I disabled the testing repositories, which downgraded my p11-kit version to 0.23.12-1, taskwarrior works like a charm now.

I should have checked the package bugtracker earlier, I am sorry. But maybe my solution helps someone else or raises awareness on a problem to come.

Thanks for the quick response @ddeimeke :-)

**Comment by jord-bh** [comment]
Thanks @MarcelPa, I hit the same problem and was only able to find the reason due to this issue.
